LookingGlass/ItemStats/ItemStats.cs

```diff
@@ -218,12 +220,18 @@ public static string GetDescription(ItemDef itemDef, ItemIndex newItemIndex, int
                                 case ItemStatsDef.MeasurementUnits.Percentage:
                                     itemDescription += $"\">{values[i] * 100:0.###}%</style>";
                                     break;
-                                case ItemStatsDef.MeasurementUnits.Health:
```

Comment:
My gut reaction to removing Health/Healing, is that it will break mods that already implement these functions, therefore impacting users that utilize both. I think it might be worth keeping the legacy enums, as to not break existing stuff.

Reply:
This should not break any legacy integrations because the enum ordering is still kept the same (enums become integers at runtime). I also made sure to test this change with a version of TooManyItems that was coded with the previous version of LookingGlass.

That being said, I can ship a quick change when I get home around 6:30 PST.

Reply:
oh, sorry. I could have probably tested it on my end before assuming. Just a lack of knowledge on my end, I think it's totally fine in that case.